BookingSystem/BookingSystem.Commands/Infrastructure/Result.cs

```diff
@@ -0,0 +1,44 @@
+﻿using BookingSystem.Commands.Properties;
+using BookingSystem.WritePersistence.WriteModels;
+
+namespace BookingSystem.Commands.Infrastructure
+{
+
+    public class Result<T>
+    {
+        public T Value { get; set; }  
+        public bool IsSuccessful { get; set; }
+        public string ErrorMessage { get; set; } 
+        //Todo: List<string> Errors{ get; set; }
+        public Result() { }
```

Comment:
I would say that default value of IsSuccessful = true and this must be presented in the constructor. Also, please, make sure that access modifiers are set correctly for all properties